data/successors/successor hails.txt

```diff
@@ -0,0 +1,394 @@
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Successor civilian
+
+phrase "successor friendly hail" # Wary hails
+	word
+		`Art thou the "noble visitor" of whom the Houses speak?`
+		`Art thou the "honored guest" the Houses do parade?`
+		"Art thou the fiendish creature that hails from distant lands?"
+		"Art thou the strange-ish traveler that hails from distant stars?"
+		"Art thou the alien visitor within our noble lands?"
+		"Art thou the creature'f history returned to seek the present?"
+		"Thy ship and countenance seem strange-ish and bizarre,"
+	word
+		" / "
+	word
+		"I note thy presence but remain thee wary."
+		"If corrupted be thy soul, then prithee keep thy distance."
+		"If thou art friend to me, so I shall be to thee."
+		"What is thy goal within these stars? To wander in delight?"
+		"To whom and what thou does thou lend thy own strength?"
+		"What is thy goal within these stars? To observe or interfere?"
+		"What is thy goal within these stars? Be a trader or a traitor?"
+
+
+phrase "successor friendly hail" # Brag hails 1
+	word
+		"I never fly without"
+	word
+		" "
+	word
+		"a trusty scanner"
+		"my trusty scanner"
+		"a larger fuel tank"
+		"communications"
+		"some conversation"
+		"a stock of crab-cubes"
+		"a stock of kelp-packs"
+		"a week's supply-cache"
+	word
+		", / "
+	word
+		"I suggest thou dost the same in thine best interest."
+		"It makes the many hours pass by quickly."
+		"The dark of space is lightened by its presence."
+		"There's little point in taking off without it."
+
+phrase "successor friendly hail" # Job hails
+	word
+		"My "
+	phrase
+		"successor relation"
+	word
+		" always told me: be a "
+		" has just earned the title: "
+		" used to know a friendly "
+		" helped me find a friendly "
+		" just got married to a "
+		" told me: never trust a "
+		" used to be a skillful "
+	phrase
+		"successor occupation"
+	word
+		" / "
+	word
+		"With these long days, I wish that they were me."
+		"A noble job, and yet not one for me."
+		"It's not my place to give critique, and yet..."
+		"I hope that I will see them soon again."
+		"Hast thou met them? Do they carry any news?"
+
+phrase "successor friendly hail" # Conspiracy hails
+	word
+		"I hope that thou will keep this thought between us:"
+		"A friend of mine had told me once in secret:"
+		"I have heard from well reputed, honest lips that:"
+		"The High Houses try to hide it, but I know that:"
+		"I've learned in many years of honest pil'ting:"
+		"If thou art willing to believe me:"
+		"Doubt my words if knave'ry is thy goal, but:"
+	word
+		" / "
+	word
+		"It's the Drak who caused the end of the old empire."
+		"It's the Quarg who caused the end of the old empire."
+		"It's the Hai who caused the end of the old empire."
+		"It's the Korath who caused the end of the old empire."
```

Comment:
Which species are the Successors aware of? They live quite far from the Hai, I believe.

Reply:
The Hai _are_ pretty distant so it makes sense for the hails to be blaming the Coalition or another more proximate species instead.
Officially, the Successors's historical experimentation with jump drives makes it plausible for them to be aware of all the spacefaring species that were around ca. 0 -- 1500 CE. Of these, it's likely the Quarg, Drak, Ka'het, Coalition, and Korath that they are most familiar with (roughly in order).